Libraries/Text/Text/RCTTextCodeBlockStyle.m

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#import "RCTTextCodeBlockStyle.h"
+#import "RCTTextAttributes.h"
+
+@implementation RCTTextCodeBlockStyle
+
+- (UIColor*)hexStringToColor:(NSString *)stringToConvert
```

Comment:
Can we use Already existing mechanism for colour in RN? 
So that other ways of passing colors work too?

https://github.com/facebook/react-native/blob/d504fb4145a95691e145ef9c99da8705a113c5d0/Libraries/StyleSheet/__tests__/processColor-test.js#L20

https://github.com/facebook/react-native/blob/3e44d207a577876f3698a4a4f4d11f1af26840a2/ReactCommon/react/renderer/graphics/Color.h


https://github.com/facebook/react-native/blob/ad0c8551c2b08acfc8f9983c31fc26513823ed3e/React/Fabric/RCTConversions.h#L37